src/PropertyLabelResolver.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace Wikibase\DataModel\Services;
```

Comment:
I was thinking of Lookup, but this class does things differently to all of the Lookups we have (so doesn't quite fit)
It could go in Term though!